workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-recipe-breadcrumb/ddm-template.ftl

```diff
@@ -0,0 +1,30 @@
+<#assign
+	journalRecipeId = .vars["reserved-article-id"].data
+	taxonomyCategoryBriefs = restClient.get("/headless-delivery/v1.0/sites/${groupId}/structured-contents/by-key/${journalRecipeId}").taxonomyCategoryBriefs
```

Comment:
If we use a variable only once it is typically fine to not make a variable and just add it to where it is used. If its really helpful to define the variable though, there should be an extra line break in between these two variables, since one depends on the other

Reply:
I believe comment was addresed on this new commit